libs/designer-ui/src/lib/monitoring/retrypanel/index.tsx

```diff
@@ -20,6 +20,11 @@ export const RetryPanel: React.FC<RetryPanelProps> = ({ retryHistories, visible
     return calculateDuration(retryHistory.startTime, retryHistory.endTime);
   }, [retryHistory.endTime, retryHistory.startTime]);
 
+  useMemo(() => {
```

Comment:
Followed up offline, but I would suggest with the correct usage of useMemo, it would look like this:

`const retryHistory = useMemo(() => retryHistories[currentPage - 1], [retryHistories, currentPage]);`

![image](https://github.com/user-attachments/assets/f975cdd8-a456-4d75-a323-fbe14eedfcc9)

Reply:
Thank you so much I will make the change